perfkitbenchmarker/benchmark_spec.py

```diff
@@ -157,7 +162,10 @@ def __init__(self, benchmark_info):
       self.project = FLAGS.project
       defaults = DEFAULTS[self.cloud]
       self.zones = FLAGS.zones or [defaults[ZONE]]
-      self.image = FLAGS.image or defaults[IMAGE]
+      if FLAGS.os_type == WINDOWS:
+        self.image = FLAGS.image or defaults.get(WINDOWS_IMAGE)
```

Comment:
I think this would be a lot cleaner if each cloud provider had a {os_type: default_image} map. Currently, would setting --os_type=rhel still install Ubuntu if you don't separately override --image?